drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -27,7 +27,7 @@ <h3>{% trans "Basic Settings" %}</h3>
                         <select class="problem-mode" aria-describedby="problem-mode-description-{{id_suffix}}">
                             {% for field_value in fields.mode.values %}
                                 <option value="{{ field_value.value }}" {% if self.mode == field_value.value %}selected{% endif %}>
-                                    {% trans field_value.display_name %}
+                                    {{  field_value.display_name }}
```

Comment:
can you point to django doc/related reference where this issue has been pointed out?

Reply:
Unable to find an exact reference to that issue, But found something similar https://code.djangoproject.com/ticket/23441#no1